src/Core/WinSWCore/Util/ConfigHelper.cs

```diff
@@ -33,5 +33,17 @@ public static TimeSpan ParseTimeSpan(string v)
             { "day",    1000L * 60L * 60L * 24L },
             { "days",   1000L * 60L * 60L * 24L }
         };
+
+        public static bool YamlBoolParse(string value)
+        {
+            value = value.ToLower();
+
+            if (value.Equals("true") || value.Equals("yes") || value.Equals("on"))
```

Comment:
add check for "1" ?

Reply:
As per [Boolean Language-Independent Type for YAML™ Version 1.1 ](https://yaml.org/type/bool.html):

```
 y|Y|yes|Yes|YES|n|N|no|No|NO
|true|True|TRUE|false|False|FALSE
|on|On|ON|off|Off|OFF
```

Reply:
This was not fully addressed. I am fine with leaving it as is and creating a GitHub issue for a follow-up improvement

Reply:
I am not sure what is missing??